playwright/before_send.spec.ts

```diff
@@ -38,18 +39,20 @@ test.describe('before_send', () => {
                         return null
                     }
 
+                    if (cr.event === AUTOCAPTURE_EVENT) {
+                        return {
+                            ...cr,
+                            event: 'redacted',
+                        }
+                    }
+
                     if (cr.event === 'custom-event') {
                         counter++
                         if (counter === 2) {
                             return null
                         }
                     }
-                    if (cr.event === '$autocapture') {
-                        return {
-                            ...cr,
-                            event: 'redacted',
-                        }
-                    }
+
```

Comment:
do you remember why this change? 
this test seems to be failing now

Reply:
which may be totally unrelated ofc

Reply:
Just thought it made sense to test it earlier, but... might have messed it up? I don't understand how that could change anything, but... happy to revert

Reply:
Let's see if [`c6e4b07` (#1930)](https://github.com/PostHog/posthog-js/pull/1930/commits/c6e4b0789b01b451eefdddd8062b5035640fa466) does the trick

Reply:
Didnt fix it, will investigate what's happening later tonight, got too many PRs open rn